packages/web-components/src/components/rux-tooltip/rux-tooltip.tsx

```diff
@@ -0,0 +1,286 @@
+
+    /**
+     *  Whether or not the tooltip is open
+     */
+    @Prop({ mutable: true, reflect: true }) open: boolean = false
+
+    /**
+     *  How long it takes the tooltip to appear in milliseconds, default = 800, Overrides the css custom property --delay.
+     */
+    @Prop({ reflect: true }) delay?: number
+
+    /**
+     *  Pixel offset from trigger, default = 8
+     */
+    @Prop() offset: number = 8
+
+    /**
+     * The placement of the tooltip relative to it's slotted trigger element. Defaults to auto.
+     */
+    @Prop() placement: ExtendedPlacement = 'auto'
+
+    /**
+     * Turns disableAutoUpdate on or off which makes the tooltip move to stay in view based on scroll. Defaults to false.
+     */
+    @Prop({ reflect: true }) disableAutoUpdate: boolean = false
+
+    /**
+     * The position strategy of the tooltip, either absolute or fixed.
+     */
+    @Prop() strategy: 'absolute' | 'fixed' = 'absolute'
+
+    /**
+     * Emits when the tooltip has opened
+     */
+    @Event({ eventName: 'ruxtooltipopened' })
+    ruxTooltipOpened!: EventEmitter
+
+    /**
+     * Emits when the tooltip has closed.
+     */
+    @Event({ eventName: 'ruxtooltipclosed' })
+    ruxTooltipClosed!: EventEmitter
+
+    @Watch('delay')
+    handleDelay() {
+        //check to see if the delay prop can be converted to a number. If not, revert to default time.
+        !isNaN(Number(this.delay)) &&
+            this.el.style.setProperty('--delay', `${Number(this.delay)}ms`)
+    }
+
+    @Watch('open')
+    handleOpen() {
+        if (this.open) {
+            this.content.style.display = 'block'
+            this._startPositioner()
+            this.ruxTooltipOpened.emit()
+        } else {
+            this.content.style.display = ''
+            this._stopPositioner()
+            this.ruxTooltipClosed.emit()
+        }
+    }
+
+    /**
+     * Opens the tooltip and returns true.
+     */
+    @Method()
+    async show() {
+        if (this.open) {
+            return this.open
+        } else this.open = true
+        return this.open
+    }
+
+    /**
+     * Closes the tooltip and returns false.
+     */
+    @Method()
+    async hide() {
+        if (!this.open) {
+            return !this.open
+        } else this.open = false
+        return !this.open
+    }
+
+    connectedCallback() {
+        this._handleSlotChange = this._handleSlotChange.bind(this)
+        this._handleTooltipShow = this._handleTooltipShow.bind(this)
+        this._handleTooltipHide = this._handleTooltipHide.bind(this)
+        this.handleDelay = this.handleDelay.bind(this)
+        this.handleOpen = this.handleOpen.bind(this)
+    }
+    disconnectedCallback() {
+        this.el!.shadowRoot!.removeEventListener(
+            'slotchange',
+            this._handleSlotChange
+        )
+    }
+    componentWillLoad() {
+        this._handleSlotChange()
+    }
+
+    componentDidLoad() {
+        this.handleOpen()
```

Comment:
Yeah makes sense since the watcher is handling that, in that case handleDelay also does not need to be there for the same reason. I removed them both.

Reply:
When I was messing around with this, I removed both (handleOpen and handleDelay) and removing the handleDelay made it so that the delay prop didn't override the animation delay. ie, <rux-tooltip delay="200"> would still be 800. Does it still work for you? I was probably just looking at the wrong test